Library/Formula/svtplay-dl.rb

```diff
@@ -1,20 +1,35 @@
 class SvtplayDl < Formula
   desc "Download videos from http://svtplay.se"
-  homepage "https://github.com/spaam/svtplay-dl"
-  url "https://github.com/spaam/svtplay-dl/archive/0.10.2015.08.24.tar.gz"
-  sha256 "0dcd2c6862ad0ed00a2b782f6484f005431de3229360e1797810f7adf16aedd9"
+  homepage "https://svtplay-dl.se"
+  url "https://pypi.python.org/packages/source/s/svtplay-dl/svtplay-dl-0.20.2015.11.29.tar.gz"
+  sha256 "7f93734da5c80387f354ccfa6cdd73af7a54978ac65ca65b1a29c801ac6dd685"
 
   bottle :unneeded
```

Comment:
Can you remove this line so a bottle gets generated (as we have a few resources now)? Thanks!